### ISSUE TITLE
Does not plot categorical variables with text values

### ISSUE THREAD

**Issue by aliabbasjp** [ayush1997/visualize_ML#2]
How can we plot categorical variables with text values?

**Comment by ayush1997** [comment]
Hi @aliabbasjp 
Thanks! for this question
So to plot categorical variables they must be encoded to numerals.
if you are using Sklearn [this](http://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.LabelEncoder.html) will help you encode them.
Auto label encode feature would be added in coming releases.

Hope this works!
Thanks
